testsuite/features/support/lavanda.rb

```diff
@@ -47,42 +83,60 @@ def hostname
     @in_hostname
   end
 
+  ##
+  # It raises an exception if the hostname is empty, otherwise it returns the hostname.
   def full_hostname
     raise 'empty hostname, something wrong' if @in_full_hostname.empty?
     @in_full_hostname
   end
 
+  ##
+  # It raises an exception if the private_ip is empty, otherwise it returns the private_ip.
   def private_ip
     raise 'empty private_ip, something wrong' if @in_private_ip.empty?
     @in_private_ip
   end
 
+  ##
+  # It returns the public IP address of the machine.
   def public_ip
     raise 'empty public_ip, something wrong' if @in_public_ip.empty?
     @in_public_ip
   end
 
+  ##
+  # It raises an error if the private interface is empty
   def private_interface
     raise 'empty private_interface, something wrong' if @in_private_interface.empty?
     @in_private_interface
   end
 
+  ##
+  # It raises an error if the public interface is empty
   def public_interface
     raise 'empty public_interface, something wrong' if @in_public_interface.empty?
     @in_public_interface
   end
 
+  ##
+  # It raises an exception if the `@in_os_family` variable is empty
   def os_family
     raise 'empty os_family, something wrong' if @in_os_family.empty?
     @in_os_family
   end
 
+  ##
+  # It raises an error if the os_version is empty.
   def os_version
     raise 'empty os_version, something wrong' if @in_os_version.empty?
     @in_os_version
   end
 
-  # run functions
+  ##
+  # It runs a command, and returns the output, error, and exit code.
+  #
+  # Args:
+  #   cmd: The command to run
```

Comment:
```suggestion
  #   cmd: The command to run.
  #   separated_results: Whether the results should be stored separately. Defaults to false.
  #   check_errors: Whether to check for errors or not. Defaults to true.
  #   timeout: The timeout to be used, in seconds. Defaults to 250 or the value of the DEFAULT_TIMEOUT environment variable.
  #   user: The user to be used to run the command. Defaults to root.
  #   successcodes: An array with the values to be accepted as success codes from the command run.
  #   buffer_size: The maximum buffer size in bytes. Defaults to 65536.
  #   verbose: Whether to log the output of the command in case of success. Defaults to false.
```